src/mbgl/map/transform_state.hpp

```diff
@@ -109,7 +110,7 @@ class TransformState {
     double x = 0, y = 0;
     double angle = 0;
     double scale = 1;
-    double altitude = 1.5;
+    double fov = 0.6435011087932844;
```

Comment:
[This comment](https://github.com/mapbox/mapbox-gl-native/blob/a6b1773a44d4ef086eb30967cb041cb64312650a/platform/darwin/src/MGLGeometry.mm#L8-L10) should also be updated to point to this field.